recipes/kerberos_init.rb

```diff
@@ -27,8 +27,8 @@
   Chef::Log.info("Secure Hadoop Enabled: Kerberos Realm '#{node['krb5']['krb5_conf']['realms']['default_realm']}'")
   secure_hadoop_enabled = true
 
-  # Create users for services
-  %w(hadoop hbase hdfs hive mapred spark yarn zookeeper).each do |u|
+  # Create users for services not in hadoop::default
```

Comment:
Truthfully, this is probably something that should be handled in `krb5_utils` cookbook's `generate_keytabs` recipe. These are only being created so that the users/groups exist prior to that cookbook being run from this recipe.